sets/sets.go

```diff
@@ -0,0 +1,25 @@
+package sets
+
+import (
+	"github.com/greymatter-io/golangz/sorting"
+)
+
+//Makes a "real" set from an array. A "real" set is an array without duplicates based upon the given equality predicate.
```

Comment:
Just one minor thing but for all comments like `//` it's generally good style to add a space between the `//` and your first character

Reply:
thanks. I added spaces and merged.